### DISCUSSION TITLE
Help with installation of the bot

### DISCUSSION THREAD

**Discussion by Claker-s** [Kyrluckechuck/TFT-Bot#192] [Q&A]
Im not going to lie i dont understand much of coding , just wanted to download a bot , however when i type "cd pip install -r requirements.txt" in the command prompt i get "the system cannot find the specified path" error.

**Comment by Kyrluckechuck** [comment]
Hey @Claker-s,

I think you might be combining two commands!
You need to `cd` into the directory you cloned using `git`, or if you downloaded the release folder, then the folder in question.
It would be whichever the folder name would be, so something like:
`cd TFT-Bot`, and then once in the directory you can run the `pip install` command!

**Comment by keyarz** [comment]
I got this error, how to fix it?

C:\Users\xxx\Desktop\TFT-Bot-main>py tft.py
Traceback (most recent call last):
  File "C:\Users\xxx\Desktop\TFT-Bot-main\tft.py", line 12, in <module>
    import keyboard
ModuleNotFoundError: No module named 'keyboard'

**Reply by Kyrluckechuck** [reply]
Hi there! While you can run this in ways other than what our guide suggests, we would recommend following our methods as we can't support others.

If that's the case, ensure you are installing and setting up your virtual environment (please see the guide for this in our README/main repo page), activate said environment, install any packages needed, and then start the bot!

If anything about the guide isn't clear, please do let us know though!

**Reply by keyarz** [reply]
yeah, i installed keyboard, Tesseract-OCR as well, but the bot just couldn't fire up, i tried to copy the keyboard folder into the bot folder, and it asks for another module, it just can't load the modules i don't know how to fix this

i ran the pip install -r requirements.txt successfully and even update for the packages
but it failed at py tft.py (asking for modules like keyboard etc.)

**Reply by Kyrluckechuck** [reply]
Odd, you definitely shouldn't be copying any folders as that can lead to havoc (and they weren't designed to be used like that) 🤔 

I realize now that I had not released the "virtual environment" changes I referenced in my response, so please check the updated README and attempt that now if you're able.

Could you also verify that `pip` and `py` are using the same version of Python? I'm wondering if maybe you have multiple versions installed and they are not using the same version

**Reply by keyarz** [reply]
hey, thanks for the help, I successfully run the bot after uninstall and reinstall python, also with the venv command :)
Guess it's because I had 2 version of pythons installed, one from .exe and another one from Microsoft store lol

btw, i saw these warnings while `pip install -r requirements.txt`

```
Collecting colorama>=0.4.5 (from pylint==2.17.4->-r requirements.txt (line 11))
  Using cached colorama-0.4.6-py2.py3-none-any.whl (25 kB)
WARNING: isort 5.12.0 does not provide the extra 'pipfile_deprecated_finder'
WARNING: isort 5.12.0 does not provide the extra 'requirements_deprecated_finder'
```
